src/components/TrackedGamesList.js

```diff
@@ -0,0 +1,70 @@
+import styled, { keyframes } from 'styled-components';
+import PropTypes from 'prop-types';
+
+TrackedGamesList.propTypes = {
+  nameOfGame: PropTypes.array
+};
+
+export default function TrackedGamesList({ nameOfGame }) {
+  return (
+    <StyledList role="list">
+      {nameOfGame.map((name, index) => (
+        <Card key={index}>
+          <CardContent>{name}</CardContent>
+        </Card>
+      ))}
+    </StyledList>
+  );
+}
+
+const spin = keyframes`
+0% {background-position: top center;}
+100% {background-position: bottom center;}
+`;
+
+const Card = styled.div`
+  width: 250px;
+  height: 45px;
+  position: relative;
+  z-index: 2;
+  border-radius: 20px;
+  margin: 60px 0;
+
+  &::before,
+  ::after {
+    content: '';
+    width: 260px;
```

Comment:
Did you intentionally change the size here (250 vs 260 and 45 vs 55)?

Reply:
Yes i did.